v4/mobile/react-native/FastLink/patches/react-native+0.62.2.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm b/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
+index 18b0e09..6e676f9 100644
+--- a/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
++++ b/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
+@@ -768,7 +768,7 @@ - (void)registerExtraLazyModules
+ #endif
+ }
+ 
+-- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<id<RCTBridgeModule>> *)modules
++- (NSArray<RCTModuleData *> *)_initializeModules:(NSArray<Class> *)modules
+                                withDispatchGroup:(dispatch_group_t)dispatchGroup
+                                 lazilyDiscovered:(BOOL)lazilyDiscovered
+ {
+diff --git a/node_modules/react-native/ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm b/node_modules/react-native/ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm
+index 841f925..338c9b5 100644
+--- a/node_modules/react-native/ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm
++++ b/node_modules/react-native/ReactCommon/turbomodule/core/platform/ios/RCTTurboModuleManager.mm
+@@ -304,7 +304,7 @@ - (void)notifyAboutTurboModuleSetup:(const char *)name
+             @"%@ has no setter or ivar for its bridge, which is not "
+              "permitted. You must either @synthesize the bridge property, "
+              "or provide your own setter method.",
+-            RCTBridgeModuleNameForClass(strongModule));
++            RCTBridgeModuleNameForClass(Class(strongModule)));
```

Comment:
Fixes this build error:

<img width="534" alt="Screenshot 2024-07-15 at 15 11 55" src="https://github.com/user-attachments/assets/dd5b7992-aa3f-4481-b408-4f19d6354744">